MapboxCoreNavigation/MGLMapboxEvents.swift

```diff
@@ -0,0 +1,39 @@
+import Mapbox
+import Polyline
+
+extension MGLMapboxEvents {
+    class func addDefaultEvents(routeProgress: RouteProgress, sessionIdentifier: UUID) -> [String: Any] {
+        var modifiedEventDictionary: [String: Any] = [:]
+    
+        modifiedEventDictionary["platform"] = UIDevice.current.systemName
+        modifiedEventDictionary["operatingSystemVersion"] = "\(UIDevice.current.systemName)-\(UIDevice.current.systemVersion)"
+        modifiedEventDictionary["sdkIdentifier"] = "mapbox-navigation-ios"
```

Comment:
This gets tricky. We could pass the event object along in the didreroute method and then have UI send off the event. But it's impossible for core to know whether UI is actually used. I think we might have to punt on this for now.